AmazonChimeSDKDemo/AmazonChimeSDKDemo/JoiningViewController.swift

```diff
@@ -29,26 +31,41 @@ class JoiningViewController: UIViewController, UITextFieldDelegate {
         meetingIdTextField.delegate = self
         nameTextField.delegate = self
 
+        callKitOptionsPicker.delegate = self
+        callKitOptionsPicker.dataSource = self
+
         setupHideKeyboardOnTap()
         versionLabel.text = "amazon-chime-sdk-ios@\(Versioning.sdkVersion())"
     }
 
     override func viewWillAppear(_: Bool) {
-        setJoinButtons(isEnabled: true)
+        callKitOptionsPicker.selectRow(0, inComponent: 0, animated: false)
+        enableJoinButton(isEnabled: true)
```

Comment:
The function is not necessary. I think the button will be enabled by default, and I do not see when it can be set disabled.

Reply:
we can put `joinButton.isEnabled = isEnabled` here and delete the func.